src/time_series_formats.jl

```diff
@@ -13,16 +12,24 @@ Return a TimeArray from a CSV file.
 
 Pass component_name when the file does not have the component name in a column header.
 """
-function read_time_series(file_path::AbstractString, component_name = nothing; kwargs...)
-    if !isfile(file_path)
-        msg = "TimeSeries file doesn't exist : $file_path"
+function read_time_series(metadata::TimeSeriesFileMetadata; kwargs...)
```

Comment:
It appears that the function `get_step_time` is no longer used.  Can all its methods and references in comments be deleted?